wiring/inc/spark_wiring_ble.h

```diff
@@ -386,7 +386,8 @@ class BleCharacteristic {
     ssize_t getValue(String& str) const;
 
     template<typename T>
-    ssize_t getValue(T* val) const {
+    typename std::enable_if<std::is_integral<T>::value, ssize_t>::type
```

Comment:
Same comment as in the other PR: https://github.com/particle-iot/device-os/pull/1901#discussion_r321124445

Reply:
And also here https://github.com/particle-iot/device-os/pull/1902/files#diff-75b7ed256204b5221da93d699fee0429R401